src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery balls = new Lottery();
```

Comment:
Lottery is a service for the provision of random balls. It's not balls itself. It's same as to have `MaternityHospital babies` variable
```suggestion
        Lottery lottery = new Lottery();
```